### PR TITLE
Add blog expanding DDEV maintainer team

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

New blog - Expanding DDEV Maintainer Team

### CODE REVIEW

File: src/content/blog/expanding-ddev-maintainer-team.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+**Randy is not forever**: Although I love DDEV and its community we are all finite, so the time will come that I either temporarily or permanently can’t carry on. That means we need a strong maintainer presence besides me.  (In fact in the short term I plan to be bike touring in Patagonia in December 2023 and January 2024 with limited access to the internet.)
+
+**The community and its needs are growing**: More CMSs have adopted DDEV as their go-to local development environment. That means we have new and different users with new and different needs. What fun! So great! But this will eventually strain our current abilities to support.
+
+### What is DDEV’s Current Funding Situation and What are the Goals?
+
+**Current Funding**:
+
+- [**Platform.sh](http://Platform.sh)** pays Randy’s salary and provides benefits to him as an employee. That is an amazing benefit for this community and goes a long way! THANK YOU!
```
Comment (stasadev) [inline src/content/blog/expanding-ddev-maintainer-team.md:46]:
```suggestion
- **[Platform.sh](http://Platform.sh)** pays Randy’s salary and provides benefits to him as an employee. That is an amazing benefit for this community and goes a long way! THANK YOU!
```

File: src/content/blog/expanding-ddev-maintainer-team.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+- **Major sponsors [Tag1](https://tag1.com/), [1xInternet](https://1xinternet.de/), and [i-gelb](https://i-gelb.net/)** together account for USD$2000/month in funding. THANK YOU!
+- **Many other agencies and individuals** via GitHub Sponsors account for about USD$1500. THANK YOU!
+
+**Funding Goal**: Our goal is salary for a full-time paid maintainer, estimated at about USD$10,000/month or USD$120,000/year. Thanks you we’re already about 35% of the way there!
+
+### What is the Current Maintainership Situation?
+
+Currently **[Randy Fay](https://github.com/rfay), [Simon Gilli](https://github.com/gilbertsoft) and [Stas Zuk](https://github.com/stasadev)** have maintainer privileges. Simon has made enormous contributions over the years and knows how to handle most maintainer roles. He has been periodically paid by the DDEV Foundation for his work, but his available time is sometimes spotty. Stas is new to the role after many, many important contributions. Financial arrangements are pending.
```
Comment (stasadev) [inline src/content/blog/expanding-ddev-maintainer-team.md:54]:
```suggestion
Currently **[Randy Fay](https://github.com/rfay), [Simon Gilli](https://github.com/gilbertsoft) and [Stas Zhuk](https://github.com/stasadev)** have maintainer privileges. Simon has made enormous contributions over the years and knows how to handle most maintainer roles. He has been periodically paid by the DDEV Foundation for his work, but his available time is sometimes spotty. Stas is new to the role after many, many important contributions. Financial arrangements are pending.
```

**Review by stasadev — CHANGES_REQUESTED** [review]
Looks good! The only thing I noticed was the broken link style and typo.

File: src/content/blog/expanding-ddev-maintainer-team.md (hunk 2 above)
Comment (rfay) [reply src/content/blog/expanding-ddev-maintainer-team.md]: Glad you caught that. Sorry for the typo.

**Comment by rfay** [comment]
Thanks for the review!

**Pull request merged**